OpenQA/Isotovideo/Utils.pm

```diff
@@ -307,7 +307,7 @@ sub checkout_git_refspec ($dir, $refspec_variable) {
     }
     my $hash = calculate_git_hash($dir);
     my $url = git_remote_url($dir);
-    bmwqemu::diag "git url in '$dir': $url";
+    bmwqemu::diag "git url in '$dir': $url ";
```

Comment:
```suggestion
    bmwqemu::diag "git url in '$dir': $url";
```

Reply:
it needs a literal whitespace. with out it adds `<span>`
![Uploading image.png…]()